### PR TITLE
Adding SSH in the FAQ for windows

### PR DESCRIPTION
**Pull request by HananAskarim**
Preview URL is https://raw.githack.com/HananAskarim/open-source/feature/adding-SSH/en_ID/#!pages/faq.md

### CODE REVIEW

File: en_ID/main.css [hunk 1]
```diff
@@ -1,99 +1,98 @@
-
 @import url('https://fonts.googleapis.com/css?family=Muli&display=swap');
 
 * {
-    margin: 0;
```
Comment (empeje) [inline en_ID/main.css:5]: Please don't create unnecessary changes like removing space like this.

File: en_ID/pages/faq.md [hunk 2]
```diff
@@ -3,147 +3,220 @@
 ## Virtual Internship
 
 ### Do Pull requests need to be merged first to be counted valid?
-* Pull requests need to be merge except one in the Step 3.
+
+- Pull requests need to be merge except one in the Step 3.
```
Comment (empeje) [inline en_ID/pages/faq.md:7]: Also, please don't change small thing like changing a `*` to `-`

**Review by empeje — CHANGES_REQUESTED** [review]

File: en_ID/main.css (hunk 1 above)
Comment (HananAskarim) [reply en_ID/main.css:5]: Sorry, I won't repeat the same mistakes again.

**Comment by empeje** [comment]
You don't need to create a new PR to make a revision.
